### ISSUE TITLE
Running with `. git-split-file.sh` froze my terminal

### ISSUE THREAD

**Issue by chambm** [Potherca/git-split-file#1]
The first time I tried to run the script I did it with `. git-split-file.sh` (I'm not enough of a Linux user to remember the difference between `bash git-split-file.sh` and `. git-split-file.sh` - I usually try one or the other to see what works).

That gave me this output:
```
Errors occurred:

 This script expects four command-line arguments

sed: can't read bash: No such file or directory
```

And then I can't escape back to the prompt, despite a flurry of control commands:
```
^C
^C^C^C^C^C
^Z
^C^C^C^C^C

sdfkljsd
ls
^C^C^C

^C
^C^C^C^C^C^C^C
^Z^Z^Z^\^\^\
```

Anyway thanks for the excellent script!

**Comment by Potherca** [comment]
Hi! Thanks for the compliment.

The correct way to call the script is `bash git-split-file.sh`. Using `. bash git-split-file.sh` will "source" the file. This means the content of the script will be loaded into the active shell.

Usually this shouldn't cause any (unwanted) side-effects, however this script is still in need of some cleanup. That's where the unwanted behaviour comes from.

I'm looking into cleaning up the script in the coming day or two, I'll report back here.

**Comment by Potherca** [comment]
Fixed in release [`v0.1.1`](https://github.com/potherca-bash/git-split-file/releases/tag/v0.1.1).
